example/src/Shared/SectionContacts.tsx

```diff
@@ -0,0 +1,212 @@
+      { name: 'Samuel Freeman', number: 5523948094 },
+      { name: 'Irene Garza', number: 2077694008 },
+      { name: 'Devante Alvarez', number: 9897002645 },
+      { name: 'Sydney Floyd', number: 6462897254 },
+      { name: 'Toni Dixon', number: 3775448213 },
+      { name: 'Anastasia Spencer', number: 4548212752 },
+      { name: 'Reid Cortez', number: 6668056507 },
+      { name: 'Ramon Duncan', number: 8889157751 },
+      { name: 'Kenny Moreno', number: 5748219540 },
+      { name: 'Shelby Craig', number: 9473708675 },
+      { name: 'Jordyn Brewer', number: 7552277991 },
+      { name: 'Tanya Walker', number: 4308189657 },
+      { name: 'Nolan Figueroa', number: 9173443776 },
+      { name: 'Sophia Gibbs', number: 6435942770 },
+      { name: 'Vincent Sandoval', number: 2606111495 },
+    ],
+  },
+]
+
+class ContactItem extends React.PureComponent<{
+  item: { name: string; number: number }
+}> {
+  render() {
+    const { item } = this.props
+
+    return (
+      <TouchableOpacity>
+        <View style={styles.item}>
+          <View style={styles.avatar}>
+            <Text style={styles.letter}>
+              {item.name.slice(0, 1).toUpperCase()}
+            </Text>
+          </View>
+          <View style={styles.details}>
+            <Text style={styles.name}>{item.name}</Text>
+            <Text style={styles.number}>{item.number}</Text>
+          </View>
+        </View>
+      </TouchableOpacity>
+    )
+  }
+}
+
+const ItemSeparator = () => <View style={styles.separator} />
+
+const renderItem = ({ item }: { item: Item }) => <ContactItem item={item} />
+
+const ListEmptyComponent = () => {
+  const { top, height } = Tabs.useHeaderMeasurements()
+  const translateY = useDerivedValue(() => {
+    return interpolate(-top.value, [0, height], [-height / 2, 0])
+  }, [height])
+
+  const stylez = useAnimatedStyle(() => {
+    return {
+      transform: [
+        {
+          translateY: translateY.value,
+        },
+      ],
+    }
+  })
+
+  return (
+    <Animated.View style={[styles.listEmpty, stylez]}>
+      <Text>Centered Empty List!</Text>
+    </Animated.View>
+  )
+}
+
+const SectionContacts: React.FC<{
+  emptyContacts?: boolean
+  nestedScrollEnabled?: boolean
+}> = ({ emptyContacts, nestedScrollEnabled }) => {
+  const [isRefreshing, startRefreshing] = useRefresh()
+
+  return (
+    <Tabs.SectionList
+      sections={emptyContacts ? [] : CONTACTS}
+      keyExtractor={(_, i) => String(i)}
+      renderItem={renderItem}
+      renderSectionHeader={({ section: { title } }) => (
+        <Text style={styles.header}>{title}</Text>
+      )}
+      ItemSeparatorComponent={ItemSeparator}
+      ListEmptyComponent={ListEmptyComponent}
+      // see https://github.com/software-mansion/react-native-reanimated/issues/1703
+      onRefresh={Platform.OS === 'ios' ? startRefreshing : undefined}
+      refreshing={Platform.OS === 'ios' ? isRefreshing : undefined}
+      nestedScrollEnabled={nestedScrollEnabled}
+    />
```

Comment:
add `stickySectionHeadersEnabled={false}` please to avoid the #136 bug

Reply:
done